code/__DEFINES/ai/ai_blackboard.dm

```diff
@@ -43,11 +43,11 @@
 ///Targetting subtrees
 #define BB_BASIC_MOB_CURRENT_TARGET "BB_basic_current_target"
 #define BB_BASIC_MOB_CURRENT_TARGET_HIDING_LOCATION "BB_basic_current_target_hiding_location"
-#define BB_TARGETTING_DATUM "targetting_datum"
+#define BB_targeting_strategy "targeting_strategy"
```

Comment:
Did you intend for these defines to be partially lowercase or did they get caught in the regex

Reply:
caught in regex I fucked it all up

Reply:
handled